### ISSUE TITLE
Missing jar on release 3.0.0

### ISSUE THREAD

**Issue by ceduliocezar** [PhilJay/MPAndroidChart#2428]
On usage instructions there is a section talking about using  mpandroidchartlibrary-version.jar as dependency, however the lastest release doesn't include a jar file like v2.2.4.

**Comment by anwar25** [comment]
Yes , I am also facing the same issue , I couldnt able to find jar for release 3.0.1

**Comment by littleneil** [comment]
![image](https://cloud.githubusercontent.com/assets/16741820/22810599/d4dc0002-ef74-11e6-9bc4-5216278c0902.png)
I find this:   https://jitpack.io/com/github/PhilJay/MPAndroidChart/v3.0.1/MPAndroidChart-v3.0.1.jar
Have a try.

**Comment by MGaetan89** [comment]
Thanks for the tip @littleneil.
This issue is a duplicate of #2358
